packages/attestation-service/src/attestation.ts

```diff
@@ -87,6 +119,107 @@ function createAttestationTextMessage(attestationCode: string) {
   return `<#> ${toBase64(attestationCode)} ${process.env.APP_SIGNATURE}`
 }
 
+async function sendSmsAndPersistAttestation(
+  attestationRequest: AttestationRequest,
+  attestationCode: string
+) {
+  const textMessage = createAttestationTextMessage(attestationCode)
+  let attestationRecord: AttestationModel | null = null
+  const AttestationTable = await getAttestationTable()
+
+  const transaction = await sequelize!.transaction()
+
+  try {
+    await AttestationTable.findOrCreate({
```

Comment:
Big Function. Can this DB interaction be moved out?

Reply:
Great idea!